.github/ISSUE_TEMPLATE/api_deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Deprecated API
+about: Track the cleanup of a deprecated API
+title: ''
+labels: 'api deprecation'
+assignees: ''
```

Comment:
Is it possible to have it automatically assign the issue to the author? If so, we may want to consider that

Reply:
Probably, right?  If you're trying to encourage accountability of cleaning up deprecations.  Otherwise in this sort of case it's kind of like throwing it into the void

Reply:
Yeah, I think that would be ideal, however I haven't been able to find anything documented on how to do that with templates. If it's not possible, I can just add a line to the template mentioning that it should be assigned to someone, and if you don't know who then assign yourself.

Reply:
I tested it out a little bit and it looks like you can't auto assign via issue templates and most likely not with issue forms either. However, I did find this GitHub Action that could work [here](https://github.com/marketplace/actions/auto-author-assign). @tylerbutler what do you think?